src/storage/StorageHttpAdminHandler.cpp

```diff
@@ -0,0 +1,125 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "storage/StorageHttpAdminHandler.h"
+#include "webservice/Common.h"
+#include "process/ProcessUtils.h"
+#include <proxygen/httpserver/RequestHandler.h>
+#include <proxygen/lib/http/ProxygenErrorEnum.h>
+#include <proxygen/httpserver/ResponseBuilder.h>
+
+namespace nebula {
+namespace storage {
+
+using proxygen::HTTPMessage;
+using proxygen::HTTPMethod;
+using proxygen::ProxygenError;
+using proxygen::UpgradeProtocol;
+using proxygen::ResponseBuilder;
+
+void StorageHttpAdminHandler::onRequest(std::unique_ptr<HTTPMessage> headers) noexcept {
+    if (headers->getMethod().value() != HTTPMethod::GET) {
+        // Unsupported method
+        err_ = HttpCode::E_UNSUPPORTED_METHOD;
+        return;
+    }
+
+    auto* space = headers->getQueryParamPtr("space");
+    if (space == nullptr) {
+        err_ = HttpCode::SUCCEEDED;
+        resp_ = "Space should not be empty. Usage: http:://ip:port/admin?space=xx&op=yy";
+        return;
+    }
+    auto* op = headers->getQueryParamPtr("op");
+    if (op == nullptr) {
+        err_ = HttpCode::SUCCEEDED;
+        resp_ = "Op should not be empty. Usage: http:://ip:port/admin?space=xx&op=yy";
+        return;
+    }
+    if (schemaMan_ == nullptr || kv_ == nullptr) {
```

Comment:
maybe can move this check to front of this function

Reply:
Good point